src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -39,6 +45,9 @@ public static long sumAll(List<Integer> numbers) {
     }
 
     public static long sumOverThreeAndDouble(List<Integer> numbers) {
-        return 0;
+        return numbers.stream()
+                .filter(number -> number > 3)
+                .map(number -> number * 2)
+                .reduce(0, Integer::sum);
```

Comment:
```suggestion
        return numbers.stream()
                .filter(number -> number > 3)
                .mapToLong(number -> number * 2)
                .reduce(0, Long::sum);
```

반환타입이 `long`이니 이렇게 해줄 수도 있겠죠 ㅎ